dist/index.js

```diff
@@ -48,10 +48,10 @@ async function runTask(ecs, clusterName, taskDefArn, waitForMinutes) {
     awsvpcConfiguration["securityGroups"] = securityGroupIds.split(',')
   }
 
-  if(assignPublicIP != ""){
+  if(assignPublicIP != "" && (subnetIds != "" || securityGroupIds != "")){
```

Comment:
I think, we should call this out in the `action.yml` file. in the description of `assignPublicIP` field. 

It would also be great if we could add a line describing the difference in networkconfiguration for different launch types.

Reply:
ok great, I made a few readme + actions.yml description changes. let me know what you think!